queries.sql

```diff
@@ -1,3 +1,80 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals
+WHERE name LIKE '%mon';
+
+SELECT name FROM animals
+WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT name FROM animals
+WHERE neutered=true AND escape_attempts<3;
+
+SELECT date_of_birth FROM animals
+WHERE name='Agumon' OR name='Pikachu';
+
+SELECT name, escape_attempts FROM animals 
+WHERE weight_kg>10.5;
+
+SELECT * FROM animals 
+WHERE neutered=true;
+
+SELECT * FROM animals 
+WHERE name <> 'Gabumon';
+
+SELECT * FROM animals 
+WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+COMMIT;
```

Comment:
Before committing, you expected to run this query ``SELECT species from animals;` in order to see your updates